.github/workflows/pr.yaml

```diff
@@ -0,0 +1,14 @@
+name: Run CI
+on: [push, pull_request]
```

Comment:
sorry, one more question. is there any reason to run this job on `push`? Seems redundant with the next job that will run for `push`.

Reply:
Yeah, that was an oversight. We don't need to run this on push.
Removed it now.